cpmonitor/templates/index.html

```diff
@@ -20,7 +21,7 @@
           </a>
           <ul class="dropdown-menu" aria-labelledby="navbarSelectCity">
             {% for city in cities %}
-              <li><a class="dropdown-item" href="#">{{ city.name }}</a></li>
+              <li><a class="dropdown-item" href="/cities/{{ city.name }}">{{ city.name }}</a></li>
```

Comment:
this link should probably be created via a django {% url %} thingy instead

Reply:
Yep. Something like `{% url "city" city.name %}` should do it. Should we add a slug to the model and use that, though?

Reply:
I think, we should clarify the URL structure for cities and tasks in another issue / PR. This can be fixed then. I will write that issue.

Reply:
Fine with me too.